data/json/snippets/newspapers.json

```diff
@@ -193,6 +206,14 @@
       {
         "id": "one_year_old_news_19",
         "text": "BIGGER BEES.  State biologists are unable to explain the emergence of bees that are two and three times normal size.  Authorities say that bees larger than golf balls are being reported across the state.  As of this time, there have been no reports of increased aggression or other behavioral changes in the oversized insects.  \"I'm glad to see the resurgence in the bee population,\" said Dr. Melissus \"Mel\" Versed, an entomologist at Tufts University.  He added, \"while Anthophila this large are unprecedented in New England, there is nothing particularly harmful about the phenomena.  Think of the honey yields!\""
+      },
+      {
+        "id": "one_year_old_news_20",
+        "text": "JAMES HAWTHORNE TWITTER FEUD.  Everyone's talking about the latest trending news on Twitter.  \"Uncanny\" tech magnate and upcoming challenger to the title of \"richest man in America\", James Hawthorne, had a protracted Twitter battle with Minnesota Democratic senatorial candidate Lester Namboney, starting over taxation laws, and running all the way down to grade school insults.  Go to page 5 for the choicest tweets."
+      },
+      {
+        "id": "one_year_old_news_21",
+        "text": "UNCANNY: SHADOWS IN PARADISE.  Uncanny, a tech development firm that has gone from nothing to a household name throughout the United States in the last two years, has been a hot topic since they entered the limelight.  We all know about James Hawthorne and his charismatic combination of sarcastic quipping and insightful observations.  Everyone has seen his tweets, and his TED Talk went viral last year.  Behind the smokescreen, though, hides labor practices that have left employees scarred and, in some cases, permanently unable to work.  Did you know that Uncanny's Portland manufacturing site tried unsuccessfully to unionize last month?  I bet you didn't.  Did you know that four union organizers disappeared prior to the vote? I bet you didn't.  Are you wondering why none of this was reported?  So am I."
```

Comment:
```suggestion
        "text": "UNCANNY: SHADOWS IN PARADISE.  Uncanny, a tech development firm that has gone from nothing to a household name throughout the United States in the last two years, has been a hot topic since they entered the limelight.  We all know about James Hawthorne and his charismatic combination of sarcastic quipping and insightful observations.  Everyone has seen his tweets, and his TED Talk went viral last year.  Behind the smokescreen, though, hides labor practices that have left employees scarred and, in some cases, permanently unable to work.  Did you know that Uncanny's Portland manufacturing site tried unsuccessfully to unionize last month?  I bet you didn't.  Did you know that four union organizers disappeared prior to the vote?  I bet you didn't.  Are you wondering why none of this was reported?  So am I."
```